test/runnable/test23.d

```diff
@@ -1366,8 +1334,6 @@ void test70()
     }
 
     static const char[0] altsep;
-    string s = std.string.format("test%spath", altsep);
-    assert(s == "testpath");
```

Comment:
Probably should confirm that this test is in phobos.

Reply:
Added it in dlang/dmd#7478 (just to be sure)